homeassistant/components/opower/strings.json

```diff
@@ -0,0 +1,28 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "utility": "Utility name",
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        }
+      },
+      "reauth_confirm": {
+        "title": "[%key:common::config_flow::title::reauth%]",
+        "description": "Enter password for {username}",
```

Comment:
We also support a read-only constant field. Will look slightly better.

Reply:
I'm not familiar with that. Can you please link to an example?

Reply:
It's the `ConstantSelector`

Reply:
You can see it in action here https://design.home-assistant.io/#components/ha-form (search for Constant)

Reply:
I changed the code to allow username to be changed in the reauth flow and having it prepopulated to the current value.

Reply:
Is there really a use case for that one? Different username but same data? And since data is synced back, should we really support that?

Reply:
PG&E doesn't let you change the username. Not sure about other utilities. I changed it to constant. People can always remove and create a new entry if they can change the username.